hbcbot/commands.py

```diff
@@ -124,6 +124,41 @@ def untappd(args):
             bid = data["response"]["beers"]["items"][0]["beer"]["bid"]
         except:
             return "Type in beer name accurately"
-        return f"https://untappd.com/beer/{bid}"
     else:
         return usage
+
+    api_url = f"https://api.untappd.com/v4/beer/info/{bid}?limit=1&client_id={untappd_client_id}&client_secret={untappd_client_secret}"
```

Comment:
```suggestion
    beer_info= f"https://api.untappd.com/v4/beer/info/{bid}?limit=1&client_id={untappd_client_id}&client_secret={untappd_client_secret}"
```

We already use api_url, so it would be nice to rename the variables to what they are doing if we're calling two